### ISSUE TITLE
Building on ubuntu 18.04

### ISSUE THREAD

**Issue by TyberiusPrime** [astroidmail/astroid#524]
Hello,

I'm trying to build on ubuntu 18.04.
I've installed the following packages:

```git scons g++ libboost1.62-all-dev libnotmuch-dev libglibmm-2.4-dev libgtkmm-3.0-dev libwebkitgtk-3.0-dev libgmime-2.6-dev libsass-dev libpeas-dev libgirepository1.0-dev``` (frow the wiki for 16.04, libboost 1.62 instead of 1.61)

In addition:
```cmake ninja-build libwebkit2gtk-4.0-dev libwebkit2gtk-4.0-37 libgmime-3.0-dev libgmime-3.0 libprotobuf-dev and libvte-2.91-dev```

But now the build fails:

``` 
cmake --build build
ninja: error: '../protobuf::protoc', needed by 'messages.pb.cc', missing and no known rule to make it
 ```

Any ideas?

**Comment by gauteh** [comment]
You also need google protobuf, and the dev version is now using webkit2
(possibly webkitgtk-4 in Ubuntu?)

ons. 11. jul. 2018 kl. 15:00 skrev Florian Finkernagel <
notifications@github.com>:

**Comment by gauteh** [comment]
https://packages.ubuntu.com/trusty/protobuf-compiler

**Comment by TyberiusPrime** [comment]
Thanks @gauteh.

webkit2: libwebkit2gtk-4.0-dev should have taken care of that.

protobuf: would have thought that 'libprotobuf-dev'  is just that, 
cmake did accept it as version >3.
Upgraded to 3.6 via this ppa https://launchpad.net/~maarten-fonville/+archive/ubuntu/protobuf
and once threw away the build folder and started over it's compiling now!

I'm not 100% certain that this was the right package, I've installed basically everything from the ppa
before recognizing I need to start the build process from scratch.

**Comment by gauteh** [comment]
Florian Finkernagel writes on July 11, 2018 16:07:
> Closed #524.

Great! Feel free to update the wiki, I'm pretty sure you should get away 
with the protobuf-compiler I linked to previously together with the C++ 
library + dev files.
